hooks/event-conversion-utils.ts

```diff
@@ -67,13 +68,33 @@ function getType (record: any): CommunityEventType {
   return 'Conference'
 }
 
-function getImage (_record: any): string {
-  const options = [
-    '/images/events/promo-finland-unconference.jpg',
-    '/images/events/promo-vermont.jpg',
-    '/images/events/promo-asia.jpg'
-  ]
-  return options[Math.floor(Math.random() * options.length)]
+function getImage (record: any): string {
+  const fallbackImage = '/images/events/no-picture.jpg'
+  const attachments = record.get(RECORD_FIELDS.image)
+  const imageAttachment = attachments && findImageAttachment(attachments)
+  const imageUrl = imageAttachment && getBestSizeUrl(imageAttachment)
+  return imageUrl || fallbackImage
+}
+
+function findImageAttachment (attachments: any[]): any|null {
+  for (const oneAttachment of attachments) {
+    const isImage = oneAttachment.type.startsWith('image')
+    if (isImage) {
+      return oneAttachment
+    }
+  }
+  return null
+}
+
+function getBestSizeUrl (imageAttachment: any): string {
+  const { thumbnails } = imageAttachment
+  const { large: largeThumbnail } = thumbnails || {}
+
+  if (!thumbnails || !largeThumbnail) {
```

Comment:
Please, try to use positive conditional as you suggested [here](https://github.com/Qiskit/qiskit.org/pull/470#discussion_r400141906)